4.3.0 grammar review stage 2 (#5020)
wazuh/wazuh-documentation: source/user-manual/capabilities/osquery.rst

Osquery
============

> Wazuh has a module that allows managing the Osquery tool from Wazuh agents to set the Osquery configuration. It collects the information generated by Osquery to send it to the manager, generating the corresponding alerts if necessary.

Reviewer: ```suggestion
Wazuh module that allows managing the Osquery tool from the Wazuh agents. It allows setting the Osquery configuration and collecting the information generated by Osquery to send it to the manager, generating the corresponding alerts if necessary.
```
